scripts/lib/api/saveImages.ts

```diff
@@ -35,13 +36,13 @@ export async function saveImages(
   await pMap(images, async (img) => {
     // The release notes images are only saved in the current version to
     // avoid having duplicate files.
-    if (img.fileName.includes("release_notes") && pkg.isHistorical()) {
+    if (pkg.isHistorical() && img.fileName.includes("release_notes")) {
```

Comment:
I inverted them because `pkg.isHistorical` is faster. Doesn't really matter though.